sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -2,22 +2,19 @@
 
 ## 2019.11 12.0.0-preview.5
 
-- [Breaking] `IPRange` is renamed to `SasIPRange`.
-- A new interface `CommonOptions` for options common to remote operations is exported. Currently, `CommonOptions` contains span options for tracing.
-- [Breaking] `Models` is no longer exported in public API surface. Instead generated model types required by the public API are explicitly re-exported. In the case where convenience layer already defined a type with conflicting name, the model type is aliased with `Model` suffix.
+- [Breaking] `IPRange` is renamed to `SasIPRange`. [PR #5551](https://github.com/Azure/azure-sdk-for-js/pull/5551)
+- A new interface `CommonOptions` for options common to remote operations is exported. Currently, `CommonOptions` contains span options for tracing. [PR #5550](https://github.com/Azure/azure-sdk-for-js/pull/5550)
```

Comment:
Sounds not easy to understand for this description:

"A new interface `CommonOptions` for options common to remote operations is exported."

Reply:
Updated as per @xirzec's suggestion.